docs/command_docs_template.md

```diff
@@ -0,0 +1,126 @@
+---
+title: SET
+description: The `SET` command in DiceDB is used to set the value of a key. If the key already holds a value, it is overwritten, regardless of its type. This is one of the most fundamental operations in DiceDB as it allows for both creating and updating key-value pairs.
+---
+<!-- description in 2 to 3 sentences, following is an example -->
+The `SET` command in DiceDB is used to set the value of a key. If the key already holds a value, it is overwritten, regardless of its type. This is one of the most fundamental operations in DiceDB as it allows for both creating and updating key-value pairs.
+
+## Syntax
+
+```bash
+SET key value [EX seconds | PX milliseconds | EXAT unix-time-seconds | PXAT unix-time-milliseconds | KEEPTTL] [NX | XX]
+```
+
+## Parameters
+<!-- add all parameters, small description, type and required -->
+
+| Parameter | Description                                                               | Type    | Required |
+|-----------|---------------------------------------------------------------------------|---------|----------|
+| `key`     | The name of the key to be set.                                            | String  | Yes      |
+|           |                                                                           |         |       |
+
+
+## Return values
```

Comment:
| Condition                                       | Return Value                                       |
|-------------------------------------------------|----------------------------------------------------|
| If key is set successfully                      | "OK"                                               |
| If `NX` is used and key already exists          | nil                                                |
| If `XX` is used and key doesn't exist           | nil                                                |


Can we add this as an example for return values?

Reply:
Yes, I ll add these.